src/pages/get-involved.js

```diff
@@ -0,0 +1,222 @@
+import React from "react";
+import { animateScroll as scroll } from 'react-scroll';
+import Layout from "../components/common/layout";
+import SEO from "../components/common/seo";
+import Button from "../components/common/button";
+import { BreakPoint, mq } from "../util/mq";
+import { css } from "@emotion/core";
+import HeroBackgroundImage from "../components/operational-review/hero-background-image";
+import linkedInImage from "../images/LinkedIn-Logos/LI-In-Bug.png";
+import NewsletterSignUp from "../components/newsletter/newsletter-sign-up";
+
+const mainCss = css`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 1.5rem 0.75rem;
+  color: var(--midnight);
+
+  h1 {
+
+    font-size: 2rem;
+    line-height: 2.75rem;
+    font-weight: 600;
+  }
+
+  h2 {
+    font-size: 1.25rem;
+    font-weight: 600;
+    margin-bottom: 14px;
+  }
+
+  p {
+    font-size: 1rem;
+  }
+
+  .heading {
+    text-align: center;
+  }
+
+  .contact-items {
+    display: grid;
+    grid-template-columns: repeat(1, 1fr);
+    grid-row-gap: 1em;
+    grid-column-gap: 2.25em;
+    grid-auto-rows: minmax(100px, auto);
+    max-width: 1024px;
```

Comment:
Use `rem` with the basis `1rem == 16px`. There are [plugins to convert them for Visual Studio](https://marketplace.visualstudio.com/items?itemName=sainoba.px-to-rem).